src/components/ContentHighlights/tests/ContentHighlights.test.jsx

```diff
@@ -5,8 +5,8 @@ import { Provider } from 'react-redux';
 import configureMockStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
 import { renderWithRouter } from '@edx/frontend-enterprise-utils';
-import { useHistory } from 'react-router';
 import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { useHistory } from 'react-router';
```

Comment:
I entered this file and saved, and it reordered the imports.

Reply:
nit: would still recommend importing from `react-router-dom` here.